src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.ts

```diff
@@ -337,10 +338,24 @@ export abstract class SimpleFindWidget extends Widget {
 		} else if (count?.resultCount) {
 			label = strings.format(NLS_MATCHES_LOCATION, count.resultIndex + 1, count?.resultCount);
 		}
+		alertFn(this._getAriaLabel(label, this.inputValue));
 		this._matchesCount.appendChild(document.createTextNode(label));
 		this._findInput?.domNode.insertAdjacentElement('afterend', this._matchesCount);
 		this._foundMatch = !!count && count.resultCount > 0;
 	}
+
+	private _getAriaLabel(label: string, searchString?: string): string {
```

Comment:
This isn't named the best, the aria label is an attribute on a DOM element whereas this is more of an "announcement" or "alert"

Reply:
Was just following suit to the editor but yeah can change that

Reply:
Thanks, just a minor thing of course but best to give it a better name